dev_cycle/terraform/input.tf

```diff
@@ -6,4 +6,15 @@ variable "key_pair_name" {}
 variable "machine_name" {}
 variable "vpc_id" {}
 variable "aws_iam_role_id" {}
-variable "ebs_kms_key_arn" {}
\ No newline at end of file
+variable "ebs_kms_key_arn" {}
+
+variable "install" {
+  type = object({
+    linuxkit_version = string
+    babashka_version = string
+  })
+  default = {
+    linuxkit_version = "master"
```

Comment:
variables could be sha commit or git tag

Reply:
awesome!